site/content/3.12/develop/http-api/indexes/multi-dimensional.md

```diff
@@ -48,24 +48,61 @@ paths:
                   type: string
                 fields:
                   description: |
-                    an array of attribute names used for each dimension. Array expansions are not allowed.
+                    An array of attribute names used for each dimension. Array expansions are not allowed.
+                  type: array
+                  items:
+                    type: string
+                fieldValueTypes:
+                  description: |
+                    must be equal to `"double"`. Currently only doubles are supported as values.
+                  type: string
+                prefixFields:
+                  description: |
+                    Requires `type` to be `"mdi-prefixed"`, and `prefixFields` needs to be set in this case.
+
+                    An array of attribute names used as search prefix. Array expansions are not allowed.
+                  type: array
+                  items:
+                    type: string
+                storedValues:
+                  description: |
+                    The optional `storedValues` attribute can contain an array of paths to additional
+                    attributes to store in the index. These additional attributes cannot be used for
+                    index lookups or for sorting, but they can be used for projections. This allows an
+                    index to fully cover more queries and avoid extra document lookups.
+                    The maximum number of attributes in `storedValues` is 32.
+
+                    It is not possible to create multiple indexes with the same `fields` attributes
+                    and uniqueness but different `storedValues` attributes. That means the value of
+                    `storedValues` is not considered by index creation calls when checking if an
+                    index is already present or needs to be created.
+
+                    In unique indexes, only the attributes in `fields` are checked for uniqueness,
+                    but the attributes in `storedValues` are not checked for their uniqueness.
+                    Non-existing attributes are stored as `null` values inside `storedValues`.
+
+                    Attributes in `storedValues` cannot overlap with attributes
+                    specified in `prefixFields` but you can have the attributes
+                    in both `storedValues` and `fields`.
```

Comment:
We could a word about why you want to have attributes in `fields` **and** `storedValues`: you can not project from `fields`. A typical use case would be to have `fields: ["x", "y"]` and `storedValues: ["x", "y", "foo"]` if you want to work with the `x` and `y` attributes of the document. If they are not in stored values, we have to load the whole document, which is a huge performance penalty.